torch/_torch_docs.py

```diff
@@ -1209,6 +1209,42 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.tensor_split,
+           r"""
+tensor_split(input, indices_or_sections, dim=0) -> List of Tensors
```

Comment:
Reviewer's note: this is consistent with torch.chunk, which also does not support an out parameter and takes three positional args.